runtime/ftplugin/help_toc.vim

```diff
@@ -0,0 +1,66 @@
+if !exists('g:no_plugin_maps')
+  function! s:show_toc() abort
+    if !exists('b:qf_toc')
```

Comment:
I'm open to this, but how would I know that the location list is the TOC for *this* help file without recomputing to make the comparison?  I assume getting the first location list item's `bufnr`, but I don't like that since quickfix is an unreliable source for state information.